[2022-06-05 20:26:18,681] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T23:25:28.506360+00:00 [queued]>
[2022-06-05 20:26:18,697] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T23:25:28.506360+00:00 [queued]>
[2022-06-05 20:26:18,697] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 20:26:18,697] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-05 20:26:18,697] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 20:26:18,719] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): populate_table> on 2022-06-05 23:25:28.506360+00:00
[2022-06-05 20:26:18,730] {standard_task_runner.py:52} INFO - Started process 164530 to run task
[2022-06-05 20:26:18,734] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'populate_table', 'scheduled__2022-06-05T23:25:28.506360+00:00', '--job-id', '995', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp4mrzljnj', '--error-file', '/tmp/tmp4tt6d59s']
[2022-06-05 20:26:18,735] {standard_task_runner.py:80} INFO - Job 995: Subtask populate_table
[2022-06-05 20:26:18,815] {task_command.py:370} INFO - Running <TaskInstance: census.populate_table scheduled__2022-06-05T23:25:28.506360+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-05 20:26:18,922] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=populate_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T23:25:28.506360+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T23:25:28.506360+00:00
[2022-06-05 20:26:19,597] {python.py:173} INFO - Done. Returned value was: None
[2022-06-05 20:26:19,622] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=populate_table, execution_date=20220605T232528, start_date=20220605T232618, end_date=20220605T232619
[2022-06-05 20:26:19,669] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-05 20:26:19,726] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
